.gitattributes

```diff
@@ -7,5 +7,5 @@
 *.html text
 *.css text
 *.inc text
-test.desc text
+*.desc text eol=lf
```

Comment:
Can we fix `test.pl` instead? I've read rumours that Strawberry Perl isn't doing a good job adapting to the OS, so maybe we should just accept that there are perl implementations that don't play ball. I believe all that it takes is replacing `chomp` in line 79 of `test.pl` by some use of `s/\R$//`.

Reply:
More concretely: Replacing `chomp @data;` by `s/\R$// for @data;` should do the trick.

Reply:
I'm happy with this if this works. I will wait for CI and if this is working, I am going to merge it.